src/Solvers/batched_tridiagonal_solver.jl

```diff
@@ -214,4 +214,4 @@ end
             ϕ[i, j, k] -= t[i, j, k+1] * ϕ[i, j, k+1]
         end
     end
-end
+end
```

Comment:
```suggestion
end

```

an empty line is nice